projects/noloan/app/src/main/java/hasadna/noloan/Util.java

```diff
@@ -0,0 +1,63 @@
+package hasadna.noloan;
+
+import com.google.firebase.firestore.auth.FirebaseAuthCredentialsProvider;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import hasadna.noloan.firebase.DbMessages;
+import hasadna.noloan.firebase.FirebaseAuthentication;
+import hasadna.noloan.firebase.FirestoreClient;
+import hasadna.noloan.protobuf.SmsProto.SmsMessage;
+
+
+// Don't know where to put this code
```

Comment:
Please mark comments that should be addressed in the review as `XXX`.
For example:
`// XXX: Don't know where to put this code`